Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -145,26 +145,22 @@ public override void Initialize()
                 return;
             }
 
-            if (!Application.isPlaying)
```

Comment:
I assume this will fix #5061 ?

Reply:
It should as this is a run time addition. :)